src/modules/hip/kernel/tensor_mean.hpp

```diff
@@ -0,0 +1,695 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+#include "reduction.hpp"
+
+// -------------------- Set 0 - Reduction Stage 2 --------------------
+__global__ void tensor_mean_grid_result_hip(Rpp32u *srcPtr,
+                                            uint xBufferLength,
+                                            float *dstPtr,
+                                            RpptROIPtr roiTensorPtrSrc)
+{
+    int id_x = hipThreadIdx_x * 8;
+    int id_z = hipBlockIdx_z;
+
+    __shared__ uint partialSum_smem[1024];                          // 8192 uints of src reduced to 1024 in a 1024 x 1 thread block
+    partialSum_smem[hipThreadIdx_x] = 0;                            // initialization of Shared to 0 using all 1024 x 1 threads
+
+    if (id_x >= xBufferLength)
+    {
+        return;
```

Comment:
You can just remove the braces for the if condition since it is a 1 line statement